tests/manage/nfs_feature/test_nfs_feature_enable_for_ODF_clusters.py

```diff
@@ -121,6 +121,11 @@ def setup_teardown(self, request):
             self.storage_cluster_obj,
         )
 
+        # ssh to test-nfs-vm
+        log.info("Login to test vm")
+        self.con = Connection(
+            "10.0.151.68", user="root", private_key=constants.SSH_PRIV_KEY, stdout=False
```

Comment:
IIUC the 10.0.151.68 is some testing VM for this purpose.
If so, I think this value should be somehow configurable and not hardcoded here. If the value will not be provided, this test should be skipped.

I am thinking that someone out of the RH will try to run the test - then it will not be valid for them.

Also I would document somewhere in test what are the prerequisites which needs to be done/installed on this VM.

I can imagine that there needs to be uploaded openshift-dev pub key in authorized_keys ,  nfs-utils package needs to be installed and so on.

Reply:
so you can create custom_config.yaml which will look like:

```yaml
ENV_DATA:
    nfs_client_ip: "10.0.151.68"
```

then provide it to run-ci via `--ocsci-conf ./custom_config.yaml`

By default in our jenkins jobs we are creating one of our custom config file, so we can make sure this value is provided in all our automation runs in Jenkins.

But if someone will run locally, they will need to create such custom config file with proper VM IP. I guess not everyone will have access to this machine and we should describe how it should be setup.

If this VM IP is not in config, then you can skip the test with link to documentation that user needs to setup such VM and set this IP in config and provide via --ocsci-conf in order to run this test.

Reply:
Addressed in latest commit. 
By default in our jenkins jobs we need to create one of our custom config file, so we can make sure
        ENV_DATA:
            nfs_client_ip: "10.0.151.68"
            nfs_client_user: "root"
            nfs_client_pwd: ""
these values are provided in all our automation runs in Jenkins.

Reply:
By default in our jenkins jobs we need to create one of our custom config file, so we can make sure
ENV_DATA:
  nfs_client_ip: "10.0.151.68"
  nfs_client_user: "root"

these values are provided in all our automation runs in Jenkins.